### PR TITLE
add Feedback form, add mailer

### PR DESCRIPTION
**Pull request by alexey-klimuk**
Email:
screenshot from 2017-06-30 17-22-06
Form:
screenshot from 2017-06-30 16-18-38
------+++++------+++++++
screenshot from 2017-06-30 16-19-01

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
[Feedback form](https://trello.com/c/8mYdFhKI/1778-feedback-form)

**Comment by alexey-klimuk** [comment]
[Feedback form](https://trello.com/c/8mYdFhKI/1778-feedback-form)

**Comment by matthewford** [comment]
@alexey-klimuk can we make the "please contact HMRC" a link to the contact page? https://www.gov.uk/government/organisations/hm-revenue-customs/contact/vat-enquiries

File: app/views/frontend_mailer/new_feedback.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<p>Hello,</p>
+
+<p>You received new feedback message:</p>
```
Comment (matthewford) [inline app/views/frontend_mailer/new_feedback.html.erb]: We can remove this, the message is sufficient

**Pull request merged**